draft-ietf-quic-multipath.md

```diff
@@ -543,22 +516,21 @@ MUST send a PATH_ABANDON frame to retire the Path ID.
 
 ### Allocating, Consuming and Retiring Connection IDs {#consume-retire-cid}
 
-Each endpoints pre-allocate a Path Identifier for each new Connection ID.
+Each endpoints pre-allocate a Path Identifier for each new connection ID.
```

Comment:
```suggestion
Endpoints first allocate unused Path Identifiers, then they issue Connection IDs for each Path Identifier.
```

Reply:
I think the first version is better, "Each endpoints pre-allocate a Path Identifier for each new connection ID." But the real process involves:

* Endpoints decide for which paths they will allocate connection identifiers, subject to both the limits imposed by "max paths" and the limits of local resource. 
* Endpoint advertise connection identifiers for these paths.
* Connection identifier values must be unique for the scope of the connection, regardless of path-id.
* The number of connection-id active for a given path must not exceed the active_connection_id_limit.

It should be clear that at any given time there will be paths for which no connection ID is advertised yet, because the peer did not yet have the time or resource to advertise them. This is not an error.

Reply:
We should rework this section in a separate PR.

Reply:
I created issue #338